composer.json

```diff
@@ -21,7 +21,7 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "psr/http-message": "^1.0",
-        "zendframework/zend-diactoros": "^1.7",
+        "zendframework/zend-diactoros": "^1.7|^2.0",
```

Comment:
Please use the `||` form, with a space on either side (consistency with other components, and closer to the language).

(I'll take care of this during merge.)